apps/fz_http/lib/fz_http_web/live/setting_live/unprivileged/account_live.ex

```diff
@@ -8,7 +8,7 @@ defmodule FzHttpWeb.SettingLive.Unprivileged.Account do
   """
   use FzHttpWeb, :live_view
 
-  alias FzHttp.Configurations, as: Conf
+  import Wrapped.Cache
```

Comment:
Imports are introducing compile-time dependencies making compilation slower: https://dashbit.co/blog/speeding-up-re-compilation-of-elixir-projects